### DISCUSSION TITLE
话说这有可能成为新的米游社吗

### DISCUSSION THREAD

**Discussion by 1492830678** [share121/inter-knot#3228] [其他]
![1710318202124369fc757ccebee6f68b1ffb5fd46be203f387e218a3d8d520165c4dece6e99a9 0](https://github.com/user-attachments/assets/c5fc0ea3-73c1-4d0b-855b-cdd34c3da662)

**Comment by xiaosheng31** [comment]
你想多了这里最多算个同人聊天的地方

**Reply by 1492830678** [reply]
唉，大本营已经是别人家的了

**Comment by Sakurakuku** [comment]
说实话，不太可能，这边门槛偏高，使用的人数肯定不高，只能说是一个小圈子

**Reply by Guessme404** [reply]
小圈子挺好的，希望一直保持

**Reply by yzysrewrwqr** [reply]
> 说实话，不太可能，人口密度偏高，使用的人数不一定高，只能说是一个小圈子

高门槛的小圈子才纯粹

**Comment by RoyXes** [comment]
一方净土（暂时）

**Comment by chillpoints** [comment]
那肯定

**Comment by kaijinti** [comment]
还是别吧，赛博生命来了这个小破地根本承受不住吧
